packages/fuels-abigen-macro/src/lib.rs

```diff
@@ -22,6 +22,19 @@ pub fn abigen(input: TokenStream) -> TokenStream {
         .into()
 }
 
+/// Abigen proc macro definition and helper functions/types for scripts
+#[proc_macro]
+pub fn script_abigen(input: TokenStream) -> TokenStream {
```

Comment:
Does `script_abigen` handle `loggedTypes`? That is, can I get parse logged values from the `Log` receipts and their types from the JSON ABI, and print them nicely?

Reply:
I don't think I currently added support for this. This PR is a first effort into bringing scritp on par with contracts. Then I will PR a big "refactor" for making the script code clearer, then will add the missing features.

Reply:
Sounds good!